gpii/node_modules/gpii-oauth2/gpii-oauth2-datastore/test/js/DbDataStoreTestEnv.js

```diff
@@ -0,0 +1,147 @@
+/*!
+Copyright 2016 OCAD university
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+/* eslint-env browser */
+/* eslint strict: ["error", "function"] */
+
+var fluid = fluid || require("infusion"),
+    gpii = fluid.registerNamespace("gpii");;
+
+(function () {
+
+    "use strict";
+
+    fluid.defaults("gpii.tests.oauth2.pouchBackedTestEnvironment", {
+        gradeNames: ["fluid.test.testEnvironment"],
+        dbViewsLocation: null,   // provided by integrators
```

Comment:
I'll adjust "provided by integrators" to "must be provided by integrators" because these two options don't have a default value and the test environment needs these values to run properly. The option with "Configurable by integrators" has a default value that the test environment can fall back to.